examples/quickstart/notebooks/quickstart.ipynb

```diff
@@ -435,7 +435,7 @@
     "@step(enable_cache=False)\n",
     "def prediction_service_loader() -> BaseService:\n",
     "    \"\"\"Load the model service of our train_evaluate_deploy_pipeline.\"\"\"\n",
-    "    repo = Repository(skip_repository_check=True)\n",
```

Comment:
why was this removed? Isn't it still nicer to suppress those warning when running the quickstart?

Reply:
same for the other change below

Reply:
you had a valid point here. I didn't want to expose users to this hidden variable, so I removed it from here. But the correct way to hide it is to detect whether a step has caching enabled _from within the step_ and only show that warning if that is the case. Which is what I did in the last commit.